config/main.py

```diff
@@ -1397,7 +1397,8 @@ def apply_patch(ctx, patch_file_path, format, dry_run, ignore_non_yang_tables, i
         # Empty case to force validate YANG model.
         if not changes_by_scope:
             asic_list = [multi_asic.DEFAULT_NAMESPACE]
-            asic_list.extend(multi_asic.get_namespace_list())
+            if multi_asic.is_multi_asic():
```

Comment:
>multi_asic [](http://example.com/codeflow?start=15&length=10)

If this is valid bugfix based on previous PR, please separate it into another smaller PR.